app/controllers/books_controller.rb

```diff
@@ -1,4 +1,33 @@
 class BooksController < ApplicationController
   def index
+    @books = Book.all
+  end
+
+  def show
+    @book = Book.find(params[:id])
+  end
+
+  def new
+    @book = Book.new
+  end
+
+  def create
+    @book = Book.create(params.require(:book).permit(:title, :author, :isbn))
```

Comment:
Wouldn't it be nicer to have this a variable so if we need to change the columns we only need to change it in one place?